### DISCUSSION TITLE
Compressor or limiter for audio

### DISCUSSION THREAD

**Discussion by Gabry2790** [EvieePy/Wavelink#306] [Ideas]
I think that a compressor or a limiter for audio is good, maybe for the equalizer that can make audio distorted (sorry for my english)

**Comment by chillymosh** [comment]
This is not something that would be implemented in Wavelink. 
What you are suggesting would need to be implemented on Lavalink as one of their filters.
Wavelink is simply a wrapper library for Lavalink.

**Reply by Gabry2790** [reply]
Oh ok
